native/c/zowex.cpp

```diff
@@ -2004,11 +2004,12 @@ int handle_uss_write(ZCLIResult result)
   auto *etag_opt2 = result.get_option("--etag-only");
```

Comment:
In the future we may want to rename the `--etag-only` option, since we now show more than just the etag 😋